src/sage/groups/libgap_mixin.py

```diff
@@ -963,3 +964,166 @@ def is_isomorphic(self, H):
             (False, False, False)
         """
         return self.gap().IsomorphismGroups(H.gap()) != libgap.fail
+
+
+@cached_method
```

Comment:
This isn't a method. However, I don't think this should be cached. If something should be cached, it should be the method attached to the group.